lib/actions/transform.js

```diff
@@ -58,7 +57,7 @@ async function processAction(msg, cfg) {
   }
 
   this.logger.info('1 message will be send to the next step');
-  this.logger.info(JSON.stringify(result));
+  this.logger.info('result is: ', JSON.stringify(result));
```

Comment:
It's helpful for this comment to be different than line 24 so you can distinguish them in the log. Since you are using the logger, the line number will probably be there anyways. So this is really a small thing.

Reply:
Appreciate the small things!